app/components/UI/Input.js

```diff
@@ -42,9 +48,21 @@ function mapFocusBorderColor(props) {
       colors: { lightningOrange, superGreen }
     }
   } = props
-  return isFieldValid(fieldState) ? superGreen : lightningOrange
+
+  if (!props.highlightOnValid) {
+    return lightningOrange
+  }
+
+  return fieldState.touched && isFieldValid(fieldState) ? superGreen : lightningOrange
 }
 
+const SearchIcon = styled(Search)`
+  margin-right: -46px;
+  width: 46px;
```

Comment:
Look like `margin-right` is equal to `width` to do some kind of offset? In that case could be convenient to introduce some kind of a prop (size or smth) `<SearchIcon size={46} />` that will adjust both of this simultaneously